go.mod

```diff
@@ -9,45 +9,34 @@ require (
 	github.com/Masterminds/goutils v1.1.0 // indirect
 	github.com/Masterminds/semver v1.5.0 // indirect
 	github.com/Masterminds/sprig v2.22.0+incompatible
-	github.com/Microsoft/hcsshim v0.8.10 // indirect
 	github.com/asaskevich/govalidator v0.0.0-20200907205600-7a23bdc65eef // indirect
 	github.com/bmatcuk/doublestar v1.3.2
 	github.com/bndr/gojenkins v1.0.1
-	github.com/containerd/containerd v1.4.1 // indirect
-	github.com/docker/docker v1.4.2-0.20200114201811-16a3519d870b // indirect
 	github.com/elliotchance/orderedmap v1.3.0
 	github.com/evanphx/json-patch v4.9.0+incompatible
-	github.com/frankban/quicktest v1.10.0 // indirect
 	github.com/getsentry/sentry-go v0.7.0
-	github.com/ghodss/yaml v1.0.0
+	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
```

Comment:
i don't get why the hash is added here? the tag should be sufficient

Reply:
Actually there is no version v1.0.1. Only exists one tag v1.0.0. But go tidy change version to v1.0.1-0.20190212211648-25d852aebe32